README.md

```diff
@@ -82,6 +82,20 @@ See the [Reporter Interface section of the API documentation](https://github.com
 
 Looking for more examples? Check out the [examples folder](https://github.com/hapijs/good/tree/master/examples).
 
+## Reporter Type
```

Comment:
I'd just make this change in API.md, No need to have to maintain this in two places.